docs/source/command_line.rst

```diff
@@ -659,6 +659,34 @@ of the above sections.
 
        assert text is not None  # OK, check against None is allowed as a special case.
 
+
+.. option:: --strict-bytes
```

Comment:
Can you also add a short section about `strict_bytes` to `config_file.rst` as well?

Reply:
Done